app/views/pages/home/_signin.html.haml

```diff
@@ -1,11 +1,15 @@
 %h2.govuk-heading-m= t('.title')
 
-%p= t('.description')
-
-- if ENV['FEATURE_SIGN_IN_ALERT'].to_s == 'true'
+- if ENV['FEATURE_SIGN_IN_ALERT'].to_s == 'true' && !authenticated?
+  %p= t('.description')
   = link_to t('sign_in.link'), new_identifications_path, class: 'govuk-button identifications-button'
-- else
-  = simple_form_for :identifications, url: identifications_path, defaults: { input_html: { class: 'text' } }, method: :post do |f|
-    = f.submit t('sign_in.link'), class: 'govuk-button'
+  %p= t('.signin_guidance_html', link: link_to(t('.signin_link'), new_identifications_path))
+
+- elsif ENV['FEATURE_SIGN_IN_ALERT'].to_s != 'true' || authenticated?
+  %div.manage-vacancies
+    %p= t('pages.home.signed_in.description') + (authenticated? ? current_school.name : '')
```

Comment:
A very minor thing for consistency. I think we prefer to interpolate than to concatenate these translations from `en.yml`. E.g.
```
        signin_guidance_html: Don't have an account? Find out %{link} if your school is eligible.
```

Reply:
done